changes/1891.feature.md

```diff
@@ -0,0 +1,2 @@
+add validate-image-canonical and validate-image-alias cli command
+
```

Comment:
Remove the extra trailing newline here as it may destroy the changelog formatting. Also, let's start the sentence with a capital letter, as you see in [the changelog](https://github.com/lablup/backend.ai/blob/main/CHANGELOG.md).